tests/benchmark_multiple_processes_all_to_all.py

```diff
@@ -0,0 +1,240 @@
+import argparse
```

Comment:
Do we want this in `tests/` or would it make sense to include in `benchmarks/`?

Reply:
I'm still thinking about it. I don't want it in tests, but I want a test there (for the UCX part only). However, a lot of the code is going to be shared and we don't have a good place currently where that common code would be visible to both. I'm not even sure the non-UCX code should live in this repo as we'll soon upstream it to OpenUCX, so it doesn't really make sense to have non-UCX code there. I'm still thinking of an appropriate place for this, if you have any suggestions, please let me know.

Reply:
Yeah that makes sense. Will think about it as well.